lib/SlackHookHandler.js

```diff
@@ -53,9 +59,15 @@ SlackHookHandler.prototype.startAndListen = function(port, tls_config) {
             });
 
             request.on("end", () => {
-                var params = qs.parse(body);
+                var isEvent = request.headers['content-type'] === 'application/json' && request.method === 'POST';
```

Comment:
If `isEvent` is true, this means it's an event from the Event API right? Could you just add a comment please?